### PR TITLE
Update references and schema

### PR DESCRIPTION
**Pull request by ds5678**

### CODE REVIEW

File: src/OnnxSharp.Test/OnnxSharp.Test.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFrameworks>net461;net5.0</TargetFrameworks>
+    <TargetFrameworks>net462;net5.0</TargetFrameworks>
```
Comment (nietras) [inline src/OnnxSharp.Test/OnnxSharp.Test.csproj]: why change target?

Comment (ds5678) [reply src/OnnxSharp.Test/OnnxSharp.Test.csproj]: It was a warning from Visual Studio. If I recall correctly, `System.Text.Json` requires at least .NET Framework 4.6.2.

Comment (nietras) [reply src/OnnxSharp.Test/OnnxSharp.Test.csproj]: 👍

**Comment by nietras** [comment]
Overall lgtm and thanks! Would you mind bumping version to 0.3.0 (I think)?

**Comment by nietras** [comment]
There seems to be some weird build errors (lack of API) on linux, macos but not windows 🤔

Test project only.

**Comment by ds5678** [comment]
I think that might have fixed it.

**Comment by nietras** [comment]
please don't remove global.json, fix version in that to latest dotnet 8 sdk

switch latestMajor to latestPatch

**Comment by ds5678** [comment]
Good now?

**Comment by ds5678** [comment]
Anything else I need to do?

**Comment by nietras** [comment]
@ds5678 no I think it should be good, I'll try to look it over tomorrow on my PC and merge if all ok, then publish new version on nuget. :)

File: src/OnnxSharp/OnnxSharp.csproj [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```
Comment (nietras) [inline src/OnnxSharp/OnnxSharp.csproj]:
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```

**Comment by nietras** [comment]
@ds5678 merged, thank you. Next is update on nuget which may not be until tomorrow though.

**Comment by nietras** [comment]
should be up on nuget now

**Pull request merged**